src/components/panel/panel.js

```diff
@@ -103,13 +103,14 @@ angular
  *     values. The panel will not open until all of the promises resolve.
  *   - `attachTo` - `{Element=}`: The element to attach the panel to. Defaults
  *     to appending to the root element of the application.
+ *   - `autoWrap` - `{boolean=}`: Whether to automatically wrap the template
```

Comment:
1. I debated adding it as an option. The reason I went with it is because it allows the user to create a completely custom component. Doing so however would mess with animations and positioning, so I would be okay with removing it. What do you think?

2. We need to be able to control two elements to get positioning correct. The first element is absolutely positioned and will cover the entire width/height of the attachTo element. The second element is the actual panel, which will be positioned according to the mdPanelPosition object. The panel is what animates, and animations would be relative to the attachTo element as well.